src/libs/antares/study/binding_constraint/BindingConstraintSaver.cpp

```diff
@@ -65,6 +65,7 @@ bool BindingConstraintSaver::saveToEnv(EnvForSaving& env, const BindingConstrain
 
     // Exporting the matrix
     env.matrixFilename.clear() << bindingConstraint->timeSeriesFileName(env);
-    return bindingConstraint->RHSTimeSeries_.saveToCSVFile(env.matrixFilename.c_str());
+    return bindingConstraint->RHSTimeSeries_.saveToCSVFile(env.matrixFilename.c_str(), 1, false, true);
```

Comment:
Don't change the save function
```suggestion
    return bindingConstraint->RHSTimeSeries_.saveToCSVFile(env.matrixFilename.c_str(), 1, false, true);
```
Modify function `BindingConstraintGroupRepository::timeSeriesWidthConsistentInGroups` to be less restrictive, accepting width of 1 (see function `checkAllElementsIdenticalOrOne`)